classes/renderables/topic_render.php

```diff
@@ -17,16 +17,20 @@ class topic_render {
     /**
      * topic_subcription_button
      * Builds new subscription button html desktop/mobile.
-     * @param string $last_reply_html html built in render.php,
-     * last replied date
-     * @param null $currently_subbed true/false if current user
+     * @param string $discussion passing the entire discussion object,
+     * using that to build the
+     * @param null $currentlysubbed true/false if current user
```

Comment:
I think this param type == null is not correct since you are not passing null to the function, correct? Any reason for adding null as a default vs something like false?